pkg/helper/taskrun/description/taskrun_description.go

```diff
@@ -80,6 +80,12 @@ No params
 {{- end }}
 {{- end }}
 
+{{decorate "timeout" ""}} {{decorate "underline bold" "Timeout\n"}}
```

Comment:
My thought would be to move this up to where, for instance, ServiceAccount, is rendered. The lower portion of the template is usually for lists, such as Steps in this case.

Reply:
I have moved timeout under Service Account